nebula-components/objects/_ui-list.scss

```diff
@@ -0,0 +1,15 @@
+@if $nb-use-ui-list {
+  .#{$nb-namespace}o-ui-list {
+    @include nb-ui-list;
+  }
+
+  .#{$nb-namespace}o-ui-list__item--divider {
+    border-bottom: 1px solid $nb-color-base-border;
+  }
+
+  .#{$nb-namespace}o-ui-list__item--divider:last-child {
```

Comment:
If we do
```
.#{$nb-namespace}o-ui-list__item + .#{$nb-namespace}o-ui-list__item {
  border-top: 1px solid $nb-color-base-border;
}
```
We won't need to remove the border on the last child.  I'd argue this is preferable as we avoid setting a style prematurely and then undoing it.

Reply:
done